tests/app/questionnaire/test_path_finder.py

```diff
@@ -100,29 +100,6 @@ def test_routing_path_with_conditional_path(self):
 
         self.assertEqual(routing_path, expected_path)
 
-    def test_routing_basic_and_conditional_path(self):
```

Comment:
How come this was removed?. The naming isn't the best.

Reply:
Thought we had discussed in tech session not needing to test the old engine anymore?

Reply:
But I agree the naming could be clearer

Reply:
Oh just realised the same tests is done below with the new schemas.